catchpoint-mixin/panels.libsonnet

```diff
@@ -0,0 +1,418 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+
+{
+  new(this):
+    {
+      local t = this.grafana.targets,
+      local stat = g.panel.stat,
+      local fieldOverride = g.panel.table.fieldOverride,
+      local alertList = g.panel.alertList,
+      local pieChart = g.panel.pieChart,
+      local barGauge = g.panel.barGauge,
+
+      // Catchpoint Overview dashboard Panels
+      topAvgLoadTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average total load time by tests',
+          targets=[t.topAvgLoadTimeTestName],
+          description='The top average total load time among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      topAvgTotalLoadTimeNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average total load time by nodes',
+          targets=[t.topAvgTotalLoadTimeNodeName],
+          description='The top average total load time among all node names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgDocumentCompletionTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average document completion time by tests',
+          targets=[t.topAvgDocumentCompletionTimeTestName],
+          description='The top average document completion time among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgDocumentCompletionTimeNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average document completion time by nodes',
+          targets=[t.topAvgDocumentCompletionTimeNodeName],
+          description='The top average document completion time among all node names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      bottomAvgRequestRatioTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Bottom average success request ratio by tests',
+          targets=[t.bottomAvgRequestSuccessRatioTestName],
+          description='The lowest average success request ratio among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withMax(1)
+        + g.panel.timeSeries.standardOptions.withMin(0)
+        + g.panel.timeSeries.standardOptions.withUnit('percentunit'),
```

Comment:
For this panel, I would suggest reducing the fill opacity to 5-10% at most

Additionally, please remove the minimum of 0 as it introduces a lot of empty space for the gradient to be distracting.

Example w. changes side by side with the node panel which has the same feedback
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/612e39ce-b9c5-4f3c-a27a-d1c2e152b02c)
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/3e819c2d-5a24-45be-ae52-eb4a6e1c8a9f)